tests/e2e/specs/add-edit-category.test.js

```diff
@@ -0,0 +1,176 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	visitAdminPage,
+	pressKeyWithModifier,
+	pressKeyTimes
+} from '@wordpress/e2e-test-utils'
+import { addQueryArgs } from '@wordpress/url';
+
+describe( 'Categories tests', () => {
+	const title = 'New Category';
+	const query = addQueryArgs( '', {
+		taxonomy: 'category',
+	} );
+
+	beforeEach( async () => {
+		/**
+		 * Delete all categories before anything
+		 * This is useful because after running more than one test
+		 * there could be existing categories
+		 */
+		await visitAdminPage( 'edit-tags.php', query );
+		await page.$( '#bulk-action-selector-top' );
+		await page.waitForSelector( '[id^=cb-select-all-]' );
+		await page.click( '[id^=cb-select-all-]' );
+		await page.select( '#bulk-action-selector-top', 'delete' );
+		await page.click( '#doaction' );
+
+		/**
+		 * Create a new category with the title 'New Category'
+		 */
+		await page.waitForSelector('#tag-name');
+		await page.focus( '#tag-name' );
+		await pressKeyWithModifier( 'primary', 'a' );
```

Comment:
Hmm, shouldn't this field be empty to start with? Or why would we need to select all here?

Reply:
My assumption is that another code could interfere on fields content.
Inspired from the login e2e https://github.com/WordPress/gutenberg/blob/f14962f283f63039be1e790f552489a1705d2928/packages/e2e-test-utils/src/login-user.js#L27

Reply:
The login form has it because browsers often autofill login forms, but that's unlikely to happen for the category form (I've never seen it, at least 😅 ). But I guess it doesn't hurt to select all anyway.

Reply:
I also noticed that sometimes, the username is filled by default (saved by browsers)